### PR TITLE
CSI Driver `version flag` is inconsistent

### PR DESCRIPTION
**Pull request by 1Shubham7**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

CSI driver `options.go` was defining its own version flag, but it conflicted 
with version flag implementation in `verflag` package). CI tests were failing because of this.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: cloud/cmd/csidriver/app/options/options.go [hunk 1]
```diff
@@ -41,6 +41,5 @@ func (o *CSIDriverOptions) Flags() (fss cliflag.NamedFlagSets) {
 	fs.StringVar(&o.DriverName, "drivername", "csidriver", "name of the driver")
 	fs.StringVar(&o.NodeID, "nodeid", "", "node id determines which node will be used to create/delete volumes")
 	fs.StringVar(&o.KubeEdgeEndpoint, "kubeedge-endpoint", "unix:///kubeedge/kubeedge.sock", "kubeedge endpoint")
-	fs.StringVar(&o.Version, "version", "dev", "version")
```
Comment (fujitatomoya) [inline cloud/cmd/csidriver/app/options/options.go:44]: Do we still need line 29 `Version`?

**Review by fujitatomoya — COMMENTED** [review]
looks good with green CI.

File: cloud/cmd/csidriver/app/options/options.go (hunk 1 above)
Comment (1Shubham7) [reply cloud/cmd/csidriver/app/options/options.go:44]: yes, we do. It is being used in [csidriver package](https://github.com/kubeedge/kubeedge/blob/e60886d889b26a4d9e8886991475bdf6ba0423f6/cloud/pkg/csidriver/server.go#L59)

**Comment by 1Shubham7** [comment]
PTAL @Shelley-BaoYue @WillardHu  : )

File: cloud/cmd/csidriver/app/server_test.go [hunk 2]
```diff
@@ -63,8 +63,8 @@ func TestNewCSIDriverCommand(t *testing.T) {
 
 	versionFlag := flags.Lookup("version")
 	assert.NotNil(versionFlag)
-	assert.Equal("version", versionFlag.Usage)
-	assert.Equal("dev", versionFlag.DefValue)
+	assert.Equal("Print version information and quit", versionFlag.Usage)
+	assert.Equal("false", versionFlag.DefValue)
```
Comment (fisherxu) [inline cloud/cmd/csidriver/app/server_test.go:67]: Looks like we've deleted the version flag, why versionFlag is still tested here?

Comment (1Shubham7) [reply cloud/cmd/csidriver/app/server_test.go:67]: actually, we just deleted the duplicate `version flag` definition, we still have the version flag and its been added [here](https://github.com/kubeedge/kubeedge/blob/e60886d889b26a4d9e8886991475bdf6ba0423f6/cloud/cmd/csidriver/app/server.go#L61). The duplicate flag was causing inconsistency during CI tests, now only one version flag definition is there.

Comment (fisherxu) [reply cloud/cmd/csidriver/app/server_test.go:67]: looks good, thanks!

**Pull request merged**